### DISCUSSION TITLE
flink cdc 同步mysql数据，时间字段经过flink cdc api处理之后时区差了8小时

### DISCUSSION THREAD

**Discussion by liu-zhen-wei** [apache/flink-cdc#1650] [Q&A]
flink cdc 同步mysql数据，时间字段经过flink cdc api处理之后时区差了8小时，使用默认的方法同步mysql的增量以及全量数据，能问下时区问题能怎么解决一下吗

**Comment by ctysf** [comment]
cdc表属性有个时区参数，设置一下就好了。 'server-time-zone' = 'Asia/Shanghai'

**Reply by liu-zhen-wei** [reply]
我用的是stream api，没有用SQL，这个配置参数没法使用

**Reply by instaer** [reply]
> 我用的是stream api，没有用SQL，这个配置参数没法使用

Please try to upgrade to flink-connector-mysql-cdc version 2.3.0

**Reply by liuxuzxx** [reply]
> cdc表属性有个时区参数，设置一下就好了。 'server-time-zone' = 'Asia/Shanghai'

这个属性是读取binlog的时候需要的。
看到还有个属性: debezium.database.connectionTimeZone: 

现在有个疑问就是：已经看到flink 从的jobManager和taskManager的容器的date返回的都是当前东8区的时间点了，但是看到打印的日志以及transform配置使用的NOW()函数，还是返回的是0时区的时间。不知道是不是flink cdc 3在哪里写死了: 0时区，又或者是有配置，但是没有找到
